Bottom: 7cc96e1654304be6bef8d35bd46de0e053801b66
Top:    7eda3bdd752c7cc4e36189102817f6b58363b82f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 02:00:01 +0000

configure.ac: Remove MINGW_CFLAGS override

Add "-mms-bitfields -mwindows" for all mingw builds and remove the option
to override via MINGW_CFLAGS.


---

diff --git a/configure.ac b/configure.ac
index 8e7a862..a293e3f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -90,8 +90,7 @@ PCB_DIR_SEPARATOR_C='/'
 case $host in
 	*-*-mingw* )
 		WIN32=yes
-		CFLAGS="$CFLAGS ${MINGW_CFLAGS:--mms-bitfields -mwindows}"
-		CPPFLAGS="$CPPFLAGS ${MINGW_CPPFLAGS}"
+		CFLAGS="$CFLAGS --mms-bitfields -mwindows"
 		;;
 
 	* )
